feature-libs/product/configurators/common/core/state/effects/configurator-basic.effect.ts

```diff
@@ -58,6 +73,21 @@ export class ConfiguratorBasicEffects {
     })
   );
 
+  collectDisabledGroupIds(
```

Comment:
Instance method needs to be after all instance fields (effects)!. I get a warning in VS code for that

Reply:
Done